core/commands/partials/vagrant_configuration_generator.rb

```diff
@@ -298,10 +294,34 @@ def get_role_description(name, product, box)
                        {}
                      end
     @ui.info("Recipe #{recipe_name}")
-    make_role_json(name, product_config, recipe_name, box)
+    [recipe_name, product_config]
   end
   # rubocop:enable Metrics/MethodLength
 
+  # Generate the role description for the specified node.
+  #
+  # @param name [String] internal name of the machine specified in the template
+  # @param products [Array<Hash>] list of parameters of products to configure from configuration file
+  # @param box [String] name of the box
+  # @return [String] pretty formatted role description in JSON format
+  def get_role_description(name, products, box)
+    errors = []
+    products_configs = {}
+    recipes_names = []
+    products.each do |product|
+      begin
+        recipe_name, product_config = make_product_config_and_recipe_name(product, box)
+      rescue RuntimeError => error
+        @ui.warning(error.message)
```

Comment:
This should be a reason to stop generating. Not just a simple warning must be here.